pipelines/jobs/configurations/jdk19_pipeline_config.groovy

```diff
@@ -171,7 +171,9 @@ class Config19 {
                 os                   : 'linux',
                 arch                 : 'riscv64',
                 configureArgs        : '--enable-dtrace --with-native-debug-symbols=none',
```

Comment:
do you know if we still need to pass `--with-native-debug-symbols=none` to the configure command? That doesn't seem quite right

Reply:
We don’t in upstream OpenJDK GHA. Not sure why it’s here in the first place though. Also, getting the debug symbols is very handy.

Reply:
ping @sxa who likey has more insight here

Reply:
From discussion on Slack, it was disabled due to the extra cost of storing/transferring the data in and out of the bots. Let's reenable it and disable it if/when it breaks.

Reply:
Now changed - resolving